### PR TITLE
First draft of 21.0.0.8-beta post

### PR DESCRIPTION
**Pull request by jakub-pomykala**

### CODE REVIEW

File: posts/2021-07-08-beta-21008-beta.adoc [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title: "In Open Liberty 21.0.0.8-beta a number of new features and improvements including; new logging format (TBASIC), The Stale Connection Identification, MicroProfile Health 3.1, Jakarta EE 9.1 convenience features and MicroProfile 4.1 improvements"
```
Comment (tevans78) [inline posts/2021-07-08-beta-21008-beta.adoc]:
```suggestion
title: "In Open Liberty 21.0.0.8-beta: MicoProfile 4.1 with MicroProfile Health 3.1; a new logging format (TBASIC); Stale Connection Identification; Jakarta EE 9.1 convenience features"
```

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 2]
```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title: "In Open Liberty 21.0.0.8-beta a number of new features and improvements including; new logging format (TBASIC), The Stale Connection Identification, MicroProfile Health 3.1, Jakarta EE 9.1 convenience features and MicroProfile 4.1 improvements"
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]:
```suggestion
title: "In Open Liberty 21.0.0.8-beta: MicoProfile 4.1 with MicroProfile Health 3.1; a new logging format (TBASIC); Stale Connection Identification; and Jakarta EE 9.1 convenience features"
```

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: In Open Liberty 21.0.0.8-beta a number of new features and improvements including; new logging format (TBASIC), The Stale Connection Identification, MicroProfile Health 3.1, Jakarta EE 9.1 convenience features and MicroProfile 4.1 improvements - OpenLiberty.io
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]: same changes as main title

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+seo-description: Open Liberty 21.0.0.8-beta brings a new logging format (`TBASIC`). The Stale Connection Identification makes it possible to provide additional congifuration for a data source, MicroProfile Health 3.1 now has unique APIs/endpoints, Jakarta EE 9.1 convenience features are included in this beta release in the `all beta` zip file and MicroProfile 4.1 improves developer experience with updates to the Health feature.
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]:
```suggestion
seo-description: Open Liberty 21.0.0.8-beta introduces MicroProfile 4.1 with updates to the MicroProfile Health feature. A new logging format (`TBASIC`) is made available. The Stale Connection Identification feature makes it possible to provide additional configuration for a data source. A couple of Jakarta EE 9.1 convenience features are also included in the `all beta` package.
```

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+blog_description: Open Liberty 21.0.0.8-beta brings a new logging format (`TBASIC`). The Stale Connection Identification makes it possible to provide additional congifuration for a data source, MicroProfile Health 3.1 now has unique APIs/endpoints, Jakarta EE 9.1 convenience features are included in this beta release in the `all beta` zip file and MicroProfile 4.1 improves developer experience with updates to the Health feature.
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]: same changes as above

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= In Open Liberty 21.0.0.8-beta a number of new features and improvements including; new logging format (TBASIC), The Stale Connection Identification, MicroProfile Health 3.1, Jakarta EE 9.1 convenience features and MicroProfile 4.1 improvements
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]: Changes similar to the title above

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 2 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 3 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 4 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 5 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 6 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+** <<mpHealth3.1, MicroProfile Health 3.1>>
+* <<tbasic, TBASIC Logging Format Beta>>
+* <<staleConnectionID, Customizing Stale Connection Identification>>
+* <<JEE9.1, Jakarta EE 9.1 convenience features>>
+
+
+[#mp]
+=== Microprofile Updates
+
+In Open Liberty 21.0.0.8-beta we have two updates: MicroProfile 4.1 improves developer experience with updates to the Health feature and MicroProfile Health 3.1 enables you to provide your own health check procedures to be invoked by Open Liberty. Find out more below.
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]:
```suggestion
In Open Liberty 21.0.0.8-beta we have two updates: MicroProfile 4.1 improves developer experience with updates to the Health feature; MicroProfile Health 3.1 enables you to provide your own health check procedures to be invoked by Open Liberty. Find out more below. 
```

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+
+When deployed application(s) are not started yet, the overall default Startup status will be DOWN, with an empty payload response until all the deployed application(s) have started. A new MicroProfile Config property (`mp.health.default.startup.empty.response=UP`) is available to change the overall default Startup check status to `UP`, during application start up.
+
+Applications are expected to provide health check procedures by implementing the HealthCheck interface with the @startup, `@Liveness`, or `@readiness` annotations. These are used by Open Liberty to verify the Startup, Liveness, or Readiness of the application, respectively. Add the logic of your health check in the `call()` method, and return the `HealthCheckResponse` object, by using the simple `up()`/`down()` methods from the API:
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]:
```suggestion
Applications are expected to provide health check procedures by implementing the HealthCheck interface with the `@Startup`, `@Liveness`, or `@readiness` annotations. These are used by Open Liberty to verify the respective Startup, Liveness, or Readiness of the application. Add the logic of your health check in the `call()` method, and return the `HealthCheckResponse` object, by using the simple `up()`/`down()` methods from the API:
```

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+To enable the new beta features in your app, add them to your `server.xml`:
+
+[source, xml]
+----
+<feature>mpHealth-3.1</feature>
+----
+
+To find out more viist:
+
+* link:https://github.com/eclipse/microprofile-health/releases/tag/3.1-RC1[MicroProfile Health Check 3.1 Release Page]
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]:
```suggestion
* link:https://github.com/eclipse/microprofile-health/releases/tag/3.1[MicroProfile Health Check 3.1 Release Page]
```

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+
+
+[source, xml]
+----
+<featureManager>
+    <feature>microProfile-4.1</feature>
+</featureManager>
+----
+
+To find out more take a look at the link:https://github.com/eclipse/microprofile/releases/tag/4.1-RC1[MicroProfile 4.1 Release].
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]:
```suggestion
To find out more take a look at the link:https://github.com/eclipse/microprofile/releases/tag/4.1-RC2[MicroProfile 4.1-RC2 Release].
```

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+
+[source, xml]
+----
+<feature>mpHealth-3.1</feature>
+----
+
+To find out more viist:
+
+* link:https://github.com/eclipse/microprofile-health/releases/tag/3.1-RC1[MicroProfile Health Check 3.1 Release Page]
+* link:https://github.com/eclipse/microprofile-health/blob/master/spec/src/main/asciidoc/release_notes.asciidoc[Release Notes]
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]:
```suggestion
* link:https://github.com/eclipse/microprofile-health/blob/3.1/spec/src/main/asciidoc/release_notes.asciidoc[Release Notes]
```

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 12]
```diff
@@ -0,0 +1,240 @@
...
+----
+[24/03/21 15:04:10:331 EDT] 00000001 FrameworkMana A   CWWKE0001I: The server defaultServer has been launched.
+[24/03/21 15:04:11:338 EDT] 00000001 FrameworkMana I   CWWKE0002I: The kernel started after 1.177 seconds
+[24/03/21 15:04:11:465 EDT] 0000003e FeatureManage I   CWWKF0007I: Feature update started.
+[24/03/21 15:04:11:635 EDT] 00000033 DropinMonitor A   CWWKZ0058I: Monitoring dropins for applications.
+----
+
+
+[#staleConnectionID]
+=== Customizing Stale Connection Identification
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc:144]: This section needs to be a bit clearer about what the "new" function is. `identifyException` is already a GA config element - https://openliberty.io/docs/21.0.0.6/reference/config/identifyException.html

File: posts/2021-07-08-logging-format-21008-beta.adoc [hunk 13]
```diff
@@ -0,0 +1,240 @@
...
+</library>
+----
+
+You can find out more on link:https://openliberty.io/docs/latest/reference/config/identifyException.html[Open Liberty Docs site].
+
+
+[#JEE9.1]
+=== Jakarta EE 9.1 convenience features
+
+Jakarta EE 9.1 convenience features are included in this beta release in the all beta zip file, but not the Jakarta EE 9 beta zip file. The 9.1 features are duplicates of the Jakarta EE 9.0 convenience features. In the future the 9.0 convenience features will be removed and only the 9.1 ones will remain. Users of the Jakarta EE 9 convenience features should look to transition to use 9.1 in preparation of 9.0 being removed.
```
Comment (tevans78) [inline posts/2021-07-08-logging-format-21008-beta.adoc]:
```suggestion
Jakarta EE 9.1 convenience features are included in this beta release in the "all beta" zip file, but not yet in the Jakarta EE 9 beta zip file. The 9.1 features are duplicates of the Jakarta EE 9.0 convenience features. In future, the 9.0 convenience features will be removed and only the 9.1 ones will remain. Users of the Jakarta EE 9 convenience features should look to transition to use 9.1 in preparation of 9.0 being removed.
```

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 7 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 8 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 9 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 10 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 11 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 12 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc:144]: Seems like Nathan will be out all week this week.

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 13 above)
Comment (jakub-pomykala) [reply posts/2021-07-08-logging-format-21008-beta.adoc]: Done

**Review by tevans78 — APPROVED** [review]

File: posts/2021-07-08-logging-format-21008-beta.adoc (hunk 12 above)
Comment (tevans78) [reply posts/2021-07-08-logging-format-21008-beta.adoc:144]: we should try to follow up with someone else then

**Pull request merged**